civihr_employee_portal/civihr_employee_portal.module

```diff
@@ -1575,101 +1576,91 @@ function civihr_employee_portal_menu() {
     'type' => MENU_CALLBACK,
   ];
 
-  $items['hr-details'] = [
```

Comment:
why was this removed?

Reply:
This menu item is not needed because it is now managed from a Panel Page.

Reply:
@powdevel I suspect this is the reason the HR Details menu item's markup is visible on login screen. Specially this part `'access arguments' => ['view my details'],`. But please investigate.

Reply:
This menu item was appearing just because I had not defined an access criteria in "My Details" page, which I already did and now it is not showing that link. By the way, that link was showing when my browser window was maximized, so I'm sure it is not there now.